web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -334,133 +427,356 @@ stored.
+- [`Unique reports`](#unique-reports) - You can uniqueing the reports by
+checking this. For more information [see](#unique-reports).
+- **Baseline**
+  - `Run / Tag Filter` - You can select one or more run names with tags. The
+  result list is restricted on the findings in these runs/tags. By selecting a
+  specific run in the *runs* view this field is filled by default.
+  When runs are stored in update mode (i.e. on the same run name), then the
+  specific runs can be tagged in order to be easier to identify them. By this
+  field you can also select the reports found during a specific run event. For
+  this you can use the cogwheel button beside each run after hovering on the
+  run to specify a tag.
+  - `Outstanding reports on a given date`**` - Filter reports that were
+  <i>DETECTED BEFORE</i> the given date and <i>NOT FIXED BEFORE</i> the given
+  date.
+- [**Compare to**](#diff-mode)
+  - `Run / Tag Filter` - Here you can select multiple runs / tags which you
+  want to compare against the baseline filter set.
+  - `Outstanding reports on a given date` - Here you can select date which
+  you want to compare against the baseline filter set.
+  - `Diff type` - Here you can set if you'd like to see the reports which
+  appear only in the **Baseline**, **Compare to** or both.
+- `File path`: You can choose a set of files to restrict the list of bug
+reports.
+- `Checker name` - If you are interested in specific type of bugs then here
+you can choose them.
+- [`Severity`](#severity-levels) - The nature of the reports is sorted
+in different severity levels. For example, a division by zero or a null pointer
+dereference is more serious than an unused variable. By this field you can
+select the reports on the given severity levels.
+- [`Latest Review Status`](#review-status) - You can select the reports
+with the given review status to check only *False positive*, *Unreviewed*, etc.
+reports.
+- [`Latest Detection Status`](#detection-status) - You can select the
+reports with the given detection status to check only *Unresolved*, *Resolved*,
+etc. reports.
+- `Analyzer name` - If you are interested in specific type of reports which
+were detected by an analyzer (Clang Tidy, CppCheck etc) then here you can
+choose them.
+- [`Source component`](https://github.com/Ericsson/codechecker/blob/master/docs/web/user_guide.md#source-components) -
+Here you can select multiple source components which are named collection of
+directories specified as directory filter.
+- `Checker message` - The static analysis tools provide a message to indicate
+the reason of a specific bug. This message is also filterable.
+- **Dates**
+  - `Detection date` - Filter reports that were detected between the given
+  interval.
+  - `Fix date` - Filter reports that were fixed between the given interval.
+- `Report hash` - Every report has a unique (hash) identifier called Report
+Identifier (RI) which can be filtered by using this input box.
```

Comment:
If we don't use abbreviation (RI) anywhere then it may be unnecessary to introduce it.